packages/app/src/debug/DebugFailedTest.cy.tsx

```diff
@@ -1,37 +1,195 @@
 import DebugFailedTest from './DebugFailedTest.vue'
+import type { TestResults } from './DebugSpec.vue'
+
+const group1 = {
+  os: {
+    id: '123',
+    name: 'Linux',
+    nameWithVersion: 'Linux Debian',
+  },
+  browser: {
+    id: '123',
+    formattedName: 'Chrome',
+    formattedNameWithVersion: 'Chrome 106',
+  },
+  groupName: 'Staging',
+  id: '123',
+}
+
+const group2 = {
+  os: {
+    id: '123',
+    name: 'Windows',
+    nameWithVersion: 'Windows 110',
+  },
+  browser: {
+    id: '123',
+    formattedName: 'Electron',
+    formattedNameWithVersion: 'Electron 106',
+  },
+  groupName: 'Production',
+  id: '456',
+}
+
+const rowElementTesting = (testResult: TestResults) => {
```

Comment:
```suggestion
const assertRowContents = (testResult: TestResults) => {
```

We can name this more inline with similar helpers in other tests.

There's a bit of conditional logic in this helper function, it would be good to have some comments explaining the rationale for what is tested in each condition.